Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,303 @@
+        /// - Returns: A new 'Map' object with added base layers.
+        private static func makeMap(spatialReference: SpatialReference, viewpointGeometry: Geometry) -> Map {
+            // Create a map with the spatial reference.
+            let map = Map(spatialReference: spatialReference)
+            map.initialViewpoint = Viewpoint(boundingGeometry: viewpointGeometry)
+            
+            // Add some base layers (counties, cities, and highways).
+            let usaLayer = ArcGISMapImageLayer(url: URL(
+                string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer"
+            )!)
+            map.basemap = Basemap(baseLayer: usaLayer)
+            
+            return map
+        }
+        
+        /// Create a graphics overlay to show the spatial reference's valid area.
+        /// - Parameter boundaryGeometry: The 'Geometry' to create the boundary graphic from.
+        /// - Returns: A new 'GraphicsOverlay' object with a boundary graphic added.
+        private static func makeBoundaryGraphicsOverlay(boundaryGeometry: Geometry) -> GraphicsOverlay {
+            let graphicsOverlay = GraphicsOverlay()
+            let lineSymbol = SimpleLineSymbol(style: .dash, color: .red, width: 5)
+            let boundaryGraphic = Graphic(geometry: boundaryGeometry, symbol: lineSymbol)
+            graphicsOverlay.addGraphic(boundaryGraphic)
+            return graphicsOverlay
+        }
+        
+        /// Create a graphics overlay for the buffer graphics.
+        /// - Returns: A new 'GraphicsOverlay' object to be used for the buffers.
+        private static func makeBufferGraphicsOverlay() -> GraphicsOverlay {
+            let graphicsOverlay = GraphicsOverlay()
+            let bufferPolygonOutlineSymbol = SimpleLineSymbol(style: .solid, color: .systemGreen, width: 3)
+            let bufferPolygonFillSymbol = SimpleFillSymbol(style: .solid, color: UIColor.yellow.withAlphaComponent(0.6), outline: bufferPolygonOutlineSymbol)
+            graphicsOverlay.renderer = SimpleRenderer(symbol: bufferPolygonFillSymbol)
+            return graphicsOverlay
+        }
+        
+        /// Create a graphics overlay for the tapped points graphics.
+        /// - Returns: A new 'GraphicsOverlay' object to be used for the tapped points.
+        private static func makeTappedPointsGraphicsOverlay() -> GraphicsOverlay {
+            let graphicsOverlay = GraphicsOverlay()
+            let circleSymbol = SimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+            graphicsOverlay.renderer = SimpleRenderer(symbol: circleSymbol)
+            return graphicsOverlay
+        }
+        
+        /// Check if a point is within the valid area of use for this sample.
+        /// - Parameter point: A 'Point' to validate.
+        /// - Returns: A 'Bool' indicating whether it is within bounds.
+        func isWithinBounds(_ point: Point) -> Bool {
```

Comment:
The method name reads a bit unnatural to me. I feel it should be similar to the geometry engine method name, sth like "boundaryContains(_:)". Will leave it open to thoughts.